src/provider/mod.rs

```diff
@@ -56,6 +56,8 @@ pub use database::Snapshot;
 const MAX_CONNECTIONS: u32 = 1024;
 const MAX_STREAMS: u64 = 10;
 const HEALTH_POLL_WAIT: Duration = Duration::from_secs(1);
+/// Default bind address for the provider.
+pub const DEFAULT_BIND_ADDR: &str = "127.0.0.1:4433";
```

Comment:
i think you can use `("127.0.0.1", 4433)` instead and that way line 89 can do `DEFAULT_BIND_ADDR.into() which is a lot nicer (i know you just copied this, but we can improve :smile: )

Reply:
Yeah the parse().unwrap() always makes me itchy. Ty :)